Externals.cfg

```diff
@@ -48,16 +48,16 @@ tag = cime6.0.45
 required = True
 
 [cmeps]
-tag = cmeps0.13.68
+tag = cmeps0.13.70-3-g2a3f929f8
 protocol = git
-repo_url = https://github.com/ESCOMP/CMEPS.git
+repo_url = https://github.com/adrifoster/CMEPS.git
 local_path = components/cmeps
 required = True
 
 [cdeps]
-tag = cdeps0.12.41
+tag = cdeps0.12.60-24-g7f47d60
```

Comment:
- [ ] As discussed, before making the final tag, these should be updated to point to tags on master / main for both cmeps and cdeps. (Just making note of it here to remind us.)